js/index.js

```diff
@@ -137,3 +138,115 @@ function creationOwlCarousel () {
             e.preventDefault();
     });
 };
+
+/**Coding section for Cards */
+if (document.readyState == 'loading') {
+    document.addEventListener('DOMContentLoaded', ready)
+} else {
+    ready();
+}
+
+function ready() {
+    let removerCartItemButtons = document.getElementsByClassName('btn-danger');
+    //console.log(removerCartItemButtons);
+    for (let i = 0; i < removerCartItemButtons.length; i++){
+        let button = removerCartItemButtons[i];
+        button.addEventListener('click', removeCartItem);
+    }
+
+    let quantityInputs = document.getElementsByClassName('cart-quantity-input');
+    for (let i = 0; i < quantityInputs.length; i++) {
+        let input = quantityInputs[i];
+        input.addEventListener('change', quantityChanged);
+    }
+
+    let addToCartButtons = document.getElementsByClassName('add-to-cart')
+    for (let i = 0; i < addToCartButtons.length; i++) {
+        let button = addToCartButtons[i];
+        button.addEventListener('click', addToCartClicked);
+    }
+
+    document.getElementsByClassName('btn-clear')[0].addEventListener('click', clearClicked)
```

Comment:
do you have multiple clear buttons? because you're taking first one after you found them